code/game/machinery/computer/arcade.dm

```diff
@@ -189,107 +248,297 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
+			temp += "<br><center><h3>[enemy_name] is getting frustrated at all your counter attacks and throws a tantrum!<center><h3>"
+			enemy_hp -= attack_amount
+
+		else if(player_stance == "counter_attack" && enemy_hp > 30 && !("smart" in current_enemy_passive))
+			temp += "<br><center><h3>[enemy_name] took the bait and allowed you to counter attack for [attack_amount * 2] damage!<center><h3>"
+			player_hp -= attack_amount
+			enemy_hp -= attack_amount * 2
+			enemy_stance = "attack"
+
+		else if(player_stance == "counter_attack" && enemy_hp <= 30)
+			temp += "<br><center><h3>[enemy_name] is getting tired of your tricks and breaks through your counter with their [chosen_weapon]!<center><h3>"
+			player_hp -= attack_amount
+			enemy_stance = "attack"
+
+		else if(!enemy_stance)
+			var/added_temp
+
+			if(rand())
+				added_temp = "you for [attack_amount + 5] damage!"
+				player_hp -= attack_amount + 5
+				enemy_stance = "attack"
+			else
+				added_temp = "the wall, breaking their skull in the process and losing [attack_amount] hp!" //[enemy_name] you have a literal dent in your skull
+				enemy_hp -= attack_amount
+				enemy_stance = "attack"
+
+			temp += "<br><center><h3>[enemy_name] grits their teeth and charge right into [added_temp]<center><h3>"
+
+	//in the case none of the previous passive triggered, Mainly here to set an enemy stance for passives that needs it like the magical passive.
+	if(!enemy_stance)
+		enemy_stance = pick("attack","defensive")
+		if(enemy_stance == "attack")
+			player_hp -= attack_amount
+			temp += "<br><center><h3>[enemy_name] attacks you for [attack_amount] points of damage with their [chosen_weapon]<center><h3>"
+			if(player_stance == "counter_attack")
+				enemy_hp -= attack_amount * 2
+				temp += "<br><center><h3>You counter [enemy_name]'s attack and deal [attack_amount * 2] points of damage!<center><h3>"
+
+		if(enemy_stance == "defensive" && enemy_mp < 15)
+			temp += "<br><center><h3>[enemy_name] take some time to get some mp back!<center><h3> "
+			enemy_mp += attack_amount
+		else if (enemy_stance == "defensive" && enemy_mp >= 15)
+			temp += "<br><center><h3>[enemy_name] quickly heal themselves for 5 hp!<center><h3> "
+			enemy_mp -= 15
+			enemy_hp += 5
+
+	//magical passive trait, recharges MP nearly every turn it's not blasting you with magic.
+	if("magical" in current_enemy_passive)
+		if(player_mp >= 50)
+			temp += "<br><center><h3>the huge amount of magical energy you have acumulated throws [enemy_name] off balance!<center><h3>"
+			enemy_mp = 0
+			LAZYREMOVE(current_enemy_passive, "magical")
+			pissed_off++
+
+		else if("smart" in current_enemy_passive && player_stance == "counter_attack" && enemy_mp > 20)
+			temp += "<br><center><h3>[enemy_name] blasts you with magic from afar for 10 points of damage before you can counter!<center><h3>"
+			player_hp -= 10
+			enemy_mp -= 20
+
+		else if(enemy_hp >= 20 && enemy_mp >= 30 && enemy_stance == "defensive")
```

Comment:
This will never be true. The maximum value enemy_mp can have at this point is 24.